new.html

```diff
@@ -0,0 +1,9 @@
+
+   <head>
```

Comment:
The Doctype, the html  and the viewport are missing please add

Reply:
Just as I added in the previous comment, always state the reasons WHY you want students to make a certain change because it gives them more insights on why your requested change is a better approach.
Here, you would agree with me, sounds a bit like a command.

An example approach would be like, **_I noticed the Doctype, HTML and Viewport are missing from your HTML document, I would suggest you include them as the Doctype specifically informs web browser about the type and version of HTML used in building the web document and Viewport helps mitigate the problem of the virtual viewport(user's view area) on narrow screen devices._**

It's long I know 😄, but I trust you get the idea. 👍